### DISCUSSION TITLE
What's the difference between a node and a device in MeshCentral?

### DISCUSSION THREAD

**Discussion by n1smithy** [Ylianst/MeshCentral#6465] [Q&A]
Hey people,

I'm trying to improve German translation. But therefor I need some explanations from the experienced users.

First of all: How does MC differ between nodes and devices?

Greetz

Schmitti

**Comment by si458** [comment]
A node is a device?
I need a little more context I'm afraid

**Reply by n1smithy** [reply]
Hehe, so we are two, yet. 

Well, it seems that MeshCentral does differentiate between nodes and devices. However the wording within the translation-list indicates this, because sometimes the word device(s) is used and sometimes it's nodes. Usually there's a reason for this like some nodes can be devices but all devices are nodes, if you know what I mean. :-/

So when I translate into German I should know if this is just a "cosmetic" difference or if this is important to differentiate between the two wordings.

**Reply by si458** [reply]
i have 100% no idea, in my eyes, nodes and devices are the same thing,
you would need to give an example of the different phrases or sentences and where they are located to understand exactly

**Reply by n1smithy** [reply]
Hm, maybe it's used because of lack of display-space on mobile devices. I don't know. Maybe I'll find out or someone who knows will answer here.
